docs/guidelines/developers/code-of-conduct.md

```diff
@@ -26,6 +26,13 @@ This document covers developers' rights, responsibilities, and rules for fair pl
 - Respect your fellow developer and communicate through any issues you have civilly.
 - Make room for other developers to get a chance to work on games they like, in their own way.
 - Do not discourage anyone from working on any game, in public or in private.
+- As players are expected to treat developers with respect, likewise it's the responsibility of a developer to ensure players receive the same amount of respect in return.
+
+## Interactions with players
+
+Interactions between players and developers may not always go as smoothly as desired. The User Code of Conduct exists to ensure players keep a healthy attitude towards developers, and likewise developers are required to do the same. Don't diminish a user's respectful feedback or interaction as harmful, and instead try to understand them and see what can be learned from it. The viewpoint of a player can be different than that of a developer with experience of how development works, so it is important they understand why certain choices are made they may disagree with.
```

Comment:
Recommendations for conciseness and clarity.
```suggestion
Interactions between players and developers may not always go smoothly. The User Code of Conduct exists to ensure players keep a healthy attitude towards developers, and likewise developers are required to do the same. Don't dismiss a user's respectful feedback as harmful or invalid. Instead, try to understand them and see what can be learned from it. A player's viewpoint can be very different than a developer's, so it is important they understand why certain choices are made they may disagree with.
```
May also want to link directly to the user CoC.